lib/router/router.js

```diff
@@ -109,6 +126,15 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && this.options.ifNoOperation && this.options.ifNoOperation.justForward === true) {
```

Comment:
You could still use the new logic of this PR if you wanted, I'd just recommend adding it to query instead via `noop` flag. That way it still leverages browser `accept`.

Feel free to re-open if you still want this feature, as a query option.